Frontend/ui-common/src/Components/PopUpModal/View.purs

```diff
@@ -116,6 +118,48 @@ view push state =
                 , visibility state.coverImageConfig.visibility
                 ]
             ]
+        ,   linearLayout
```

Comment:
Add condition to render this view here else `setYoutubePlayer` might be called unnecessarily